src/gui/wizard/owncloudwizard.cpp

```diff
@@ -19,14 +19,13 @@
 #include "theme.h"
 #include "owncloudgui.h"
 
-#include "wizard/owncloudwizard.h"
-#include "wizard/welcomepage.h"
-#include "wizard/owncloudsetuppage.h"
-#include "wizard/owncloudhttpcredspage.h"
-#include "wizard/owncloudoauthcredspage.h"
+#include "wizard/flow2authcredspage.h"
 #include "wizard/owncloudadvancedsetuppage.h"
+#include "wizard/owncloudhttpcredspage.h"
+#include "wizard/owncloudsetuppage.h"
+#include "wizard/owncloudwizard.h"
 #include "wizard/webviewpage.h"
-#include "wizard/flow2authcredspage.h"
+#include "wizard/welcomepage.h"
```

Comment:
would have been nicer not to change order here

Reply:
Sorry, I have automatic clang-format enabled, will undo this change

Reply:
done